components/style/gecko/values.rs

```diff
@@ -187,6 +189,48 @@ impl GeckoStyleCoordConvertible for Angle {
     }
 }
 
+impl GeckoStyleCoordConvertible for ExtremumLength {
+    fn to_gecko_style_coord<T: CoordDataMut>(&self, coord: &mut T) {
+        coord.set_value(CoordDataValue::Enumerated(
+            match *self {
```

Comment:
Please don't hardcode the constants, instead import them from `gecko_bindings::structs::`

https://github.com/glasserc/servo/blob/413a73f3685600f70303deb5cdee2af0b86b0442/components/style/gecko_bindings/structs_debug.rs#L508